slam-ws/src/twist_mux/CMakeLists.txt

```diff
@@ -49,9 +49,3 @@ install(DIRECTORY include/
    DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION}
    FILES_MATCHING PATTERN "*.h"
 )
-
-if (CATKIN_ENABLE_TESTING)
-    find_package(rostest REQUIRED)
-
-    add_rostest(test/system.test)
-endif()
```

Comment:
I believe this is a valid fix, but is it potentially possible to set the `CATKIN_ENABLE_TESTING` flag to `false`?